timed_proc_macros/src/lib.rs

```diff
@@ -9,9 +9,9 @@ use syn::{AttributeArgs, ItemFn};
 #[derive(Debug, FromMeta)]
 struct TracingArgs {
     #[darling(default)]
-    enabled: Option<bool>,
+    _enabled: Option<bool>,
     #[darling(default)]
-    main: Option<String>,
+    _main: Option<String>,
```

Comment:
These 2 fields are never read according to clippy, I didn't want to delete them because it'll be a breaking change. Wdyt?